### DISCUSSION TITLE
Why do I need a line break in Exception?

### DISCUSSION THREAD

**Discussion by ne-skazhu** [seleniumbase/SeleniumBase#1891] [General]
Why? If there are errors, the message is not displayed in the short test summary info
```python
def _format_message(message):
    message = "\n " + message
    return message
```

**Comment by mdmintz** [comment]
Output is optimized for log files, such as the ``basic_test_info.txt`` file that appears in the ``latest_logs/`` folder. It prevents a long and messy word wrap. Lines in the short test summary area would get truncated anyway, and usually not in an optimal location.
